modules/etargetBidAdapter.js

```diff
@@ -57,18 +58,45 @@ export const spec = {
       data: bidderRequest,
       bids: validBidRequests,
       netRevenue: netRevenue,
+      metaData: getMetaData(),
       bidder: 'etarget',
       gdpr: gdprObject
     };
 
+    function getMetaData() {
+      var mts = {};
+      var hmetas = document.getElementsByTagName('meta');
+      var wnames = ['title', 'og:title', 'description', 'og:description', 'og:url', 'base', 'keywords'];
```

Comment:
can you look in the ortb2 object for this stuff as well?

Publishers or a prebid module might be setting it directly 
https://github.com/prebid/Prebid.js/blob/585c7107645ee9f79befb9d17d2cf8c6000b0d00/modules/enrichmentFpdModule.js#L64

or via methods in #5795

Reply:
Thanks for recommendation, but this is not solving my problem. With or without my new function "getMetaData" circleci is failing tests.

Reply:
I'm aware, this is something else you should fix